gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -126,9 +130,9 @@ protected void removeToken(final String token) {
 
     try {
       aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME, token);
-      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME,token + "--max");
+      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME,token + "TOKEN_MAX_LIFETIME_POSTFIX");
     } catch (AliasServiceException e) {
-      log.failedToUpdateTokenExpiration(e);
+      log.errorAccessingTokenState(e);
```

Comment:
Why did this log message change?

Reply:
This was an error in my previous commit that @smolnar82 pointed after the commit was merged.